config/initializers/_config.rb

```diff
@@ -7,7 +7,9 @@
   basic_auth_pass: ENV.fetch('BASIC_AUTH_PASS', nil),
   minimum_version_ruby: ENV.fetch('MINIMUM_VERSION_RUBY', nil),
   minimum_version_node: ENV.fetch('MINIMUM_VERSION_NODE', nil),
-  working_dir: ENV.fetch('WORKING_DIR', nil)
+  working_dir: ENV.fetch('WORKING_DIR', nil),
+  datadog_service_name: ENV['DATADOG_SERVICE_NAME'] || 'horizon',
+  datadog_trace_agent_hostname: ENV.fetch('DATADOG_TRACE_AGENT_HOSTNAME', nil)
```

Comment:
Will be set via [k8](https://github.com/artsy/horizon/blob/main/hokusai/production.yml#L50)